static/app/views/dashboardsV2/contextMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {MouseEvent} from 'react';
+import styled from '@emotion/styled';
+import classNames from 'classnames';
+
+import DropdownMenu from 'app/components/dropdownMenu';
+import {IconEllipsis} from 'app/icons';
+
+export const ContextMenu = ({children}) => (
```

Comment:
I think it would be great if we used only named exports, but we have agreed that, for now, we will continue to use default exports in components when it is possible